resources/test/fixtures/flake8_simplify/SIM201.py

```diff
@@ -0,0 +1,16 @@
+if not a == b: #SIM201
+  pass
+
+if not a == (b + c): #SIM201
+  pass
+
+if not (a+b) == c: #SIM201
+  pass
+
+if not a != b: #NG
```

Comment:
What's `NG` stand for here?

Reply:
NG - Not Good cases.
I will remove it if it's confusing, I am just used to NG as an opposite to OK.

Reply:
I changed it to `OK`, is that wrong? Since these cases are "OK", in that they don't trigger errors? Let me now if I've misunderstood, and I'll correct in a follow-up commit.